site/guide/configuration/manage-permissions.qmd

```diff
@@ -17,7 +17,7 @@ Use permissions to streamline and secure access control within your organization
 
 ## Steps
 
-::: {.callout-important title="Be careful when modifying permissions to ensure they align with the role's intended responsibilities and access requirements."}
+::: {.callout-important title="When assigning permissions to roles, ensure they align with the role's intended responsibilities and access requirements."}
 Please note that the `Customer Admin` role has the highest level of access and as such, its permissions cannot be modified.
```

Comment:
Either a missing comma or an extra comma, depending on how comma-tose you are. (I'll see myself out.)

```suggestion
Please note that the `Customer Admin` role has the highest level of access and, as such, its permissions cannot be modified.
```